templates/common/infra/bicep/core/host/container-app.bicep

```
@@ -58,6 +89,10 @@ resource app 'Microsoft.App/containerApps@2022-03-01' = {
           }
         }
       ]
+      scale: {
+        minReplicas: containerMaxReplicas > 1 ? containerMaxReplicas : 1
+        maxReplicas: containerMinReplicas > 1 ? containerMinReplicas : 1
```

For min replicas, min value is 0, right? Should we also check for upper limits? 

```suggestion
        minReplicas: containerMaxReplicas > 0 ? containerMaxReplicas : 0
        maxReplicas: containerMinReplicas > 1 ? containerMinReplicas : 1
```

https://learn.microsoft.com/en-us/azure/container-apps/scale-app?pivots=azure-cli